src/libraries/System.Security.Cryptography.Algorithms/src/System.Security.Cryptography.Algorithms.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DefineConstants>$(DefineConstants);INTERNAL_ASYMMETRIC_IMPLEMENTATIONS</DefineConstants>
-    <NoWarn>$(NoWarn);CA5350;CA5351;CA5379;CA5384</NoWarn>
+    <NoWarn>$(NoWarn);CA5350;CA5351;CA5379;CA5384;CA1416</NoWarn>
```

Comment:
There is several types derived from HashAlgorithm or using HashAlgorithmName (SHA256, SHA1, SHA1Managed, SHA256Managed, SHA384, SHA384Managed, SHA512, SHA512Managed, IncrementalHash) and called from browser context, seems `System.Security.Cryptography.HashAlgorithmName and System.Security.Cryptography.HashAlgorithm` supported on browser cc @bartonjs

Reply:
When do we plan on addressing

> seems System.Security.Cryptography.HashAlgorithmName and System.Security.Cryptography.HashAlgorithm should be supported on browser.

?

It feels like this should have a follow up issue logged for it if we aren't addressing that problem here.

Reply:
/cc @akoeplinger

Reply:
The issue appears to be that we are generating the attribute for all of System.Security.Cryptography.Primitives which not correct.

Reply:
The attribute added assembly level, which applies to all Types/APIs/Enums etc. Do you mean we should have attributed it to lower-level like Type level? Do you mean HashAlgorithm and HashAlgorithmName are shouldn't have unsupported on browser?

Reply:
Yeah I see in the props. I mean I think assembly level is probably too aggressive on the Primitive types especially the ones that are actually supported and used on browser.

Reply:
> It feels like this should have a follow up issue logged for it if we aren't addressing that problem here.

Issue created https://github.com/dotnet/runtime/issues/43380

Reply:
Perfect. Thank you.

Reply:
This is addressed with https://github.com/dotnet/runtime/pull/43387